docs/userguide.md

```diff
@@ -0,0 +1,188 @@
+page_title: Kitematic User Guide: Intro & Overview
+page_description: Documentation that provides an overview of Kitematic and installation instructions
+page_keywords: docker, documentation, about, technology, kitematic, gui
+
+# Kitematic user guide
+
+## Overview
+
+Kitematic is an open source project built to simplify and streamline using
+Docker on a Mac or Windows (coming soon) PC. Kitematic automates the Docker
+installation and setup process and provides an intuitive graphical user
+interface (GUI) for running Docker containers.  Kitematic integrates with
+[Docker Machine](http://docs.docker.com/machine/) to provision a VirtualBox VM
+and install the Docker Engine locally on your machine.
+
+Once installed, the Kitematic GUI launches and from the home screen you will be
+presented with curated images that you can run instantly. You can search for any
+public images on Docker Hub from Kitematic just by typing in the search bar. 
+You can use the GUI to create, run and manage your containers just by clicking
+on buttons. Kitematic allows you to switch back and forth between the Docker CLI
+and the GUI.  Kitematic also automates advanced features such as managing ports
+and configuring volumes.  You can use Kitematic to change environment variables,
+stream logs, and single click terminal into your Docker container all from the
+GUI.
+
+First, if you haven't yet done so, [download and start
+Kitematic](./index.md).
+
+## Container list
+
+Kitematic lists all running and stopped containers on the left side, underneath
+the "New Container" link.
+
+The container list includes all containers, even those not started by Kitematic,
+giving you a quick over-view of the state of your Docker daemon.
+
+You can click on any container to view its logs (the output of the PID-0 process),
+restart, stop or exec `sh` in that container. See [Working with a
+container](#working-with-a-container) for more details.
+
+## Creating a new container
+
+The "New Container" page lets you search for and select from images on the Docker Hub.
+When you've found the image you want to run, you can click "Create" to pull, create,
+and run the container.
+
+![Nginx create](../assets/browse-images.png)
+
+## Working with a container
+
+If you select a non-running container, either stopped, or paused, you will be able to
+"Restart" or "Stop" the container using the icons. You can also view the entire PID-0
```

Comment:
PID-0 might be a little advanced for some users - do we refer to this as anything else? The `main process` in the container?